src/data/countryData.ts

```diff
@@ -1,7 +1,7 @@
 // The data was taken from the form react-phone-input-2
 // https://github.com/bl00mber/react-phone-input-2/blob/master/src/rawCountries.js
 
-import { CountryData } from '../types';
+import { CountryData } from "../types";
```

Comment:
Sorry to bother you again, seems like Prettier did not apply the project's config (and changed all the single quotes to double)
Can you run `pnpm run format` and `pnpm run lint` and make sure that there are no linting issues?